doc/api/perf_hooks.md

```diff
@@ -56,11 +56,15 @@ to mark specific significant moments in the Performance Timeline.
 ### `performance.measure(name, startMark, endMark)`
 <!-- YAML
 added: v8.5.0
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/32651
+    description: Make `startMark` and `endMark` parameters optional.
 -->
 
 * `name` {string}
-* `startMark` {string}
-* `endMark` {string}
+* `startMark` {string | undefined}
+* `endMark` {string | undefined}
```

Comment:
If I read the code correctly, this still throws if `undefined` is passed in explicitly, I think? You may want to change the heading here to indicate that `startMark` and `endMark` are optional instead, and/or fix that in the code